pages/lessons/projects/6.mdx

```diff
@@ -0,0 +1,796 @@
+---
+i18n: 'erc20'
+title: Your own Token with Foundry
+description:
+  Create and deploy an ERC20 Token (UNI, MKR, DAI, etc.) using the Foundry
+  toolkit.
+icons:
+  [
+    'solidity',
+    'foundry',
+    'openzeppelin',
+    'ethereum',
+    'etherscan',
+  ]
+---
+
+# Create your own Tokens using Foundry
+
+## What are we creating?
+
+At the end of this lesson you will be able to create your own ERC20 tokens, and
+you will be able to change its properties in any way you need.
+
+## Tokens, Crypto, Coins, ERC20, is it all the same?
+
+Let's begin with some context, so we all speak the same language.
+
+The Ethereum blockchain (and most EVM-compatible blockchains) has a native token
+that is used to pay for transaction costs, and also to reward validators in
+Proof of Stake, or previously miners in Proof of Work. In Ethereum that native
+token is Ether (or ETH), in Polygon its MATIC, and so on.
+
+Let's focus on the Ethereum blockchain for the sake of clarity.
+
+Every _other_ Token or "Crypto" you see, use or interact with in Ethereum is
+basically a smart contract that lets you send this tokens, receive them and
```

Comment:
"lets you send this tokens," should be "lets you send these tokens"

Reply:
Already working on commiting all these corrections John! Thanks!